pages/02.Tutorials/09.dnsaas/docs.en.md

```diff
@@ -0,0 +1,387 @@
+---
+title: 'DNS as a service tutorial'
```

Comment:
Looking at the titles of our other tutorials, I believe something like 
```suggestion
title: 'Using DNS as a Service'
```
or 
```suggestion
title: 'Get started with DNS as a Service'
```
would be better

Reply:
I like the first variant, too.